README.md

```diff
@@ -51,6 +51,9 @@ The last user in the list is "Not.Authorised@example.com." Select this user to s
 
 This authentication strategy can be disabled locally by setting ```DEV_AUTH_ENABLED=false```
 
+**NOTE:** The dev_auth authentication strategy has been enabled for local development and docker-compose by setting ```DEV_AUTH_ENABLED=true``` and ```IS_LOCAL_DOCKER_ENV=true```.
+This must never be enabled in the live (staging/production) service in it's current form as it will expose sensitive information.
```

Comment:
I tried to be a bit vague as to what would be exposed as it doesn't seem like a good idea to shout about it in the readme 😅 is it worth keeping this?